telco-examples/edge-clusters/README.md

```diff
@@ -27,7 +27,16 @@ You need to modify the following values in the `telco-edge-cluster.yaml` file:
 
 You need to modify the following folder:
 
-- `base-images` - To include inside the `SL-Micro.x86_64-6.0-Base-RT-GM2.raw` image downloaded from the SUSE Customer Center.
+- `base-images` - To include inside the raw image generated with the following Kiwi process:
```

Comment:
```suggestion
- `base-images` - To include the raw image generated by Kiwi as:
```

Reply:
content is the same, but let me change it :| 
Please ...